package.json

```diff
@@ -1751,21 +1751,20 @@
             "description": "%configuration.razor.languageServer.debug%",
             "order": 90
           },
-          "razor.trace": {
+          "razor.server.trace": {
```

Comment:
Any benefit in trying to migrate this setting using something like the [migrateOptions.ts](https://github.com/dotnet/vscode-csharp/blob/main/src/shared/migrateOptions.ts)?

Reply:
I thought about it, but figured it wasnt worth the effort since the Razor server had a bug, and didn't actually honour the old value anyway.